app-admin/logstash-bin/files/gentoo-logstash-plugin

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
Yes. I mostly mean replacing the boilerplate you needed here with the magic eselect does.